docs/collector.mssql.md

```diff
@@ -24,80 +24,80 @@ Name | Description | Type | Labels
-`wmi_mssql_availreplica_received_from_replica_bytes` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sent_to_replica_bytes` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sent_to_transport_bytes` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_initiated_flow_controls` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_flow_control_wait_seconds` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_receives_from_replica` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_resent_messages` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sends_to_replica` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_availreplica_sends_to_transport` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_bufman_background_writer_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_buffer_cache_hit_ratio` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_checkpoint_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_database_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_extension_allocated_pages` | _Not yet documented_ | counter | `instance`
-`wmi_mssql_bufman_extension_free_pages` | _Not yet documented_ | counter | `instance`
+`wmi_mssql_accessmethods_au_batch_cleanups` | The number of batches per second that were completed successfully by the background task that cleans up deferred dropped allocation units | counter | `instance`
+`wmi_mssql_accessmethods_au_cleanups` | The number of allocation units per second that were successfully dropped the background task that cleans up deferred dropped allocation units. Each allocation unit drop requires multiple batches | counter | `instance`
+`wmi_mssql_accessmethods_by_reference_lob_creates` | Count of large object (lob) values that were passed by reference. By-reference lobs are used in certain bulk operations to avoid the cost of passing them by value | counter | `instance`
+`wmi_mssql_accessmethods_by_reference_lob_uses` | Count of by-reference lob values that were used. By-reference lobs are used in certain bulk operations to avoid the cost of passing them by-value | counter | `instance`
+`wmi_mssql_accessmethods_lob_read_aheads` | Count of lob pages on which readahead was issued | counter | `instance`
+`wmi_mssql_accessmethods_column_value_pulls` | Count of column values that were pulled in-row from off-row | counter | `instance`
+`wmi_mssql_accessmethods_column_value_pushes` | Count of column values that were pushed from in-row to off-row | counter | `instance`
+`wmi_mssql_accessmethods_deferred_dropped_aus` | The number of allocation units waiting to be dropped by the background task that cleans up deferred dropped allocation units | counter | `instance`

```

Comment:
I don't have access to a MSSQL server to check this, but as this is documented as a counter, it probably is the total number of AUs ever deferred. Or it could also be that it should actually be a gauge.